### PR TITLE
Refactored python code so that it conforms to PEP8

### LINKED ISSUE #35: Refactor Idiomatic python
There are few things in code which does not align with python best practices or conventions, especially the variable and method names. We should scan the complete code and fix anything that deviates from guidelines listed in [PEP 8 -- Style Guide for Python Code
](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)

### PR DESCRIPTION
**Pull request by Rohan-cod**
Used 'autopep8' to refactor the complete python code so that it conforms to PEP8.
Fixes #35.

### CODE REVIEW

**Review by srgothi92 — COMMENTED** [review]
Thanks for this change, really appreciate all your hard work.
 One thing before we merge this, can we add a step in our GitHub actions to validate formatting, so any new PR is validated to formatting along with tests.

**Comment by Rohan-cod** [comment]
@srgothi92, should I add this validation to GitHub Actions via a new pull request from a new branch or commit to this pull request and squash both the commits?

**Comment by srgothi92** [comment]
> @srgothi92, should I add this validation to GitHub Actions via a new pull request from a new branch or commit to this pull request and squash both the commits?

We can have a separate PR that updates our GitHub actions to validate formatting and we can merge that before this PR. Once that is merged we should be able to see CI failing for other commit and this commit should be able to make CI happy. This way we can also validate our GitHub actions works.

File: vcf2fhir/common.py [hunk 1]
```diff
@@ -13,14 +13,14 @@ class _Utilities(object):
 
     """
     /**
-     * generates current date into fhir format 
+     * generates current date into fhir format
      * @return date
      */
     """
     def getFhirDate():
```
Comment (srgothi92) [inline vcf2fhir/common.py:20]: I would also like to see the naming convention being changed according to [this](https://www.python.org/dev/peps/pep-0008/#id45), here and any other place throughout the codebase.

Ideally we can also get rid of Utiltiy class and just create utility functions as discussed [here](https://stackoverflow.com/questions/19620498/how-to-create-a-utility-class-correctly). However, this suggestion is irrelevant to purpose of this PR.

**Review by srgothi92 — COMMENTED** [review]
Can you squash commits here as well. If you look at commit header and content of commit they do not match.   
If interested, you can read https://chris.beams.io/posts/git-commit/ , its an interesting blog that describes what is the best way to write commit message.

**Review by srgothi92 — APPROVED** [review]

**Comment by srgothi92** [comment]
@Rohan-cod can you fix git conflicts for this commit.

**Comment by Rohan-cod** [comment]
> @Rohan-cod can you fix git conflicts for this commit.

Sure sir, I will resolve the conflicts.

**Comment by Rohan-cod** [comment]
@srgothi92, I am trying to squash the merge commit into the first commit, but am unable to do so. I searched on the internet and asked on StackOverflow too. If possible, can you please squash and merge using the 'squash and merge' option as shown in the image below

select-squash-and-merge-from-drop-down-menu

I apologize for the inconvenience caused.

**Comment by srgothi92** [comment]
> @srgothi92, I am trying to squash the merge commit into the first commit, but am unable to do so. I searched on the internet and asked on StackOverflow too. If possible, can you please squash and merge using the 'squash and merge' option as shown in the image below

I prefer to avoid using this option particularly because the commit [message](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#merge-message-for-a-squash-merge) of squashed commit does not describe the actual purpose of commit as would be expected by the author. Can you locally squash the commit and format commit message as required.

**Comment by Rohan-cod** [comment]
Sir, the commit message after squash using the 'squash and merge option' on GitHub according to this [link](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#merge-message-for-a-squash-merge) will be the pull request title. And, the pull request title for this pull request is exactly the same as the first commit's message.

**Comment by Rohan-cod** [comment]
Actually, the problem is caused by the second [commit](https://github.com/elimuinformatics/vcf2fhir/pull/39/commits/7337602910e57d4e5308f9748a99804a54f2fb3b) in this PR. I asked about this on StackOverflow and received the following response
Screenshot 2021-04-02 at 9 14 11 PM

**Comment by srgothi92** [comment]
Is there anything that is blocking you from squash this locally ? If it is difficult and you need help, I can pull and submit a squashed commit on your fork.

**Comment by Rohan-cod** [comment]
It would be great if you can help Sir. Usually, I do the following:-

1. git rebase -i HEAD~n, n being the number of commits
2. Then I place squash in front of the commits that I want to squash.

But, in this case, the second commit is not showing up in the rebase.

**Comment by srgothi92** [comment]
Can you please close this PR.

**Pull request closed without merging**